Formula/p/pgcopydb.rb

```diff
@@ -0,0 +1,30 @@
+class Pgcopydb < Formula
+  desc "Copy a Postgres database to a target Postgres server"
+  homepage "https://github.com/dimitri/pgcopydb"
+  url "https://github.com/dimitri/pgcopydb/archive/refs/tags/v0.17.tar.gz"
+  sha256 "7ed96f7bbc0a5250f3b73252c3a88d665df9c8101a89109cde774c3505882fdf"
+  license "PostgreSQL"
+  head "https://github.com/dimitri/pgcopydb.git", branch: "main"
+
+  on_linux do
+    depends_on "gcc"
+    depends_on "bdw-gc"
+    depends_on "libpq"
+  end
+  
+  depends_on "bdw-gc" => [:build]
+  depends_on "libpq" => [:build]
+  depends_on "sphinx-doc" => [:build]
+
+  def install
+    system "make", "clean"
```

Comment:
We build in a tmpdir so nothing will exist there before.
```suggestion
```

Reply:
Removed